afl-fuzz.c

```diff
@@ -2457,6 +2457,15 @@ static u8 run_target(char** argv, u32 timeout) {
   memset(trace_bits, 0, MAP_SIZE);
   if (dsf_enabled || save_everything) { 
     memset(dsf_map, 0, dsf_len_actual * sizeof(u32));
```

Comment:
Does it make sense to simply do`memset(dsf_map, dsf_initial, ...)`? Does that address the issue you've discovered?